spec/apps/invalid-load-function/invalid-load-function.spec.js

```diff
@@ -71,4 +71,34 @@ describe(`invalid-load-function`, () => {
         expect(singleSpa.getAppStatus('invalid-load-3')).toBe(singleSpa.SKIP_BECAUSE_BROKEN);
       })
   });
+
+  it('It retries loading an application that previously failed', () => {
+    let count = 0;
+    function loadFunction() {
+      count++;
```

Comment:
The first time the load function is called we want to error, the second time it will succeed